src/circuits/joinsplit.tcc

```diff
@@ -254,7 +256,10 @@ public:
                 unpacked_inputs[i].insert(
                     unpacked_inputs[i].end(),
                     input_nullifiers[i]->bits.rbegin() +
-                        (HashT::get_digest_len() - FieldT::capacity()),
+                        safe_subtraction(
```

Comment:
Might make sense to factor some of these out into a `const size_t` and save some lines.

Reply:
Yes indeed, and more importantly you save useless function calls all the way down ;)
Changed in last commit